app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -639,7 +639,6 @@ private void changeSurfaceLayout(int videoWidth, int videoHeight, int videoVisib
         double ar, vw;
```

Comment:
The declaration of `vw` could be moved to line 645 since it is only used to compute the `ar` value there.

Reply:
Fixed, I've also removed some more junk code from this file.
Some properties were set but never read, other were read but never set.